commit/factory.go

```diff
@@ -148,11 +152,21 @@ func (p *PluginFactory) NewReportingPlugin(ctx context.Context, config ocr3types
 		oracleIDToP2PID[commontypes.OracleID(oracleID)] = node.P2pID
 	}
 
+	// map types to the facade.
+	readers := make(map[cciptypes.ChainSelector]contractreader.ContractReaderFacade, len(p.contractReaders))
+	for chain, cr := range p.contractReaders {
+		chainID, err1 := sel.ChainIdFromSelector(uint64(chain))
```

Comment:
This appears to be deprecated:
> // Deprecated, this only supports EVM chains, use the chain agnostic `GetChainIDFromSelector` instead

Reply:
You are right, let me bump chain selectors there